evoting/frontend/package.json

```diff
@@ -1,111 +1,61 @@
 {
   "name": "evoting",
-  "version": "1.10.0",
-  "description": "A Vue.js project",
-  "author": "DEDIS",
+  "version": "0.1.1",
   "private": true,
   "scripts": {
-    "dev": "genversion src/version.js && webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
-    "start": "npm run dev",
-    "unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run",
-    "e2e": "node test/e2e/runner.js",
-    "test": "npm run unit && npm run e2e",
```

Comment:
has the code for the tests been removed, or only its running? 
it'd be good to still be able to run them, especially given the changes :)